tx_handler/migration_tx_handler_test.go

```diff
@@ -29,9 +30,9 @@ func TestMigrationTxHandler(t *testing.T) {
 	migrationTx1 := mockMessageTx(t, uint32(1), origin, origin)
 
 	migrationFuncs := map[int32]MigrationFunc{
-		1: func(ctx *migrations.MigrationContext) error { return nil },
-		2: func(ctx *migrations.MigrationContext) error { return nil },
-		3: func(ctx *migrations.MigrationContext) error { return nil },
+		1: func(ctx *migrations.MigrationContext, parameters []byte) error { return nil },
+		2: func(ctx *migrations.MigrationContext, parameters []byte) error { return nil },
+		3: func(ctx *migrations.MigrationContext, parameters []byte) error { return nil },
```

Comment:
Update one of these functions to expect the parameters to match a particular value, and update the tests below to pass that value.